.github/workflows/createNewVersion.yml

```diff
@@ -70,6 +55,11 @@ jobs:
           # This is a workaround to allow pushes to a protected branch
           token: ${{ secrets.OS_BOTIFY_COMMIT_TOKEN }}
 
+      - name: Validate actor
```

Comment:
The only difference between this PR and the previous one is that this step was moved to happen after the checkout. Trying to call a local action before actions/checkout doesn't work